opencencus-check-listener/types.go

```diff
@@ -1,41 +1,48 @@
-package gosundheit
+package opencencus
 
 import (
 	"context"
+	"log"
+	"strconv"
+
 	"go.opencensus.io/stats"
 	"go.opencensus.io/stats/view"
 	"go.opencensus.io/tag"
-	"log"
-	"strconv"
+)
+
+const (
+	// ValAllChecks is the value used for the check tags when tagging all tests
+	ValAllChecks = "all_checks"
 )
 
 var (
-	keyCheck, _        = tag.NewKey("check")
-	keyCheckPassing, _ = tag.NewKey("check_passing")
+	keyCheck, _          = tag.NewKey("check")
+	keyCheckPassing, _   = tag.NewKey("check_passing")
+	keyClassification, _ = tag.NewKey("classification")
 
 	mCheckStatus   = stats.Int64("health/status", "An health status (0/1 for fail/pass)", "pass/fail")
 	mCheckDuration = stats.Float64("health/execute_time", "The time it took to execute a checks in ms", "ms")
 
 	// ViewCheckExecutionTime is the checks execution time aggregation tagged by check name
 	ViewCheckExecutionTime = &view.View{
 		Measure:     mCheckDuration,
-		TagKeys:     []tag.Key{keyCheck},
+		TagKeys:     []tag.Key{keyCheck, keyClassification},
```

Comment:
do we get extra rows when classification is empty?

Reply:
No, as the context will not add that tag